packages/core/src/components.d.ts

```diff
@@ -825,8 +827,10 @@ declare namespace LocalJSX {
         "value"?: IonTypes.IonSelect['value'];
     }
     interface AtomStepsModal {
```

Comment:
question: I know that this file is generated automatically but is weird `interface AtomStepsModal` generating twice, do you know what?

Reply:
i think that is the way stencil create those interfaces, because if u see, all the component interfaces are created twice, one with the types being in the way on the component, other being all props setted as optional

Reply:
i do not know the why, but i think it's duplicated interface because namespace. It must create a representation of the AtomModal in a specific namespace, type: global, HTML, etc...